test/jdk/java/security/testlibrary/SimpleOCSPServer.java

```diff
@@ -292,6 +292,13 @@ public synchronized void stop() {
         }
     }
 
+    public synchronized void shutdownNow() {
```

Comment:
This is the tricky one how to make shutdownNow precise and effective. It is not clear for the jtreg timeout where the server start has stalled in the launch of the service thread. Is it stuck in the the executors thread pool, or has the thread been created an is on the Ready to Run queue of the OS due to heavy load on system. The server thread doesn't handle a Thread.interrupt, and it inferred that this mechanism used by the ExecutorService to terminate a task then  the try within the run method would need to handle InterruptedException and invoke some shutdown e.g. close the server socket etc.  Or will that be taken care by try with resources and autoclave. We known or can reasonably assume that the server thread has not invoked the count latch countdown method.  One could theorise that if the server thread got bumped after the bind but during the log call (as its IO) then there will be potential be some server socket left open.  I think it would be worthwhile to emulate where the server thread doesn't start in timely manner by inserting a Thread.sleep in the run method sleep > 10 seconds and see if the restart works effectively and that multiple server sockets are not left open
In short need to look at the shotdown processing more closely to ensure that it does the necessary and appropriate cleanup for each retry iteration